examples/org-website/list_sample.go

```diff
@@ -1,46 +1,51 @@
+//go:build ignore
```

Comment:
what is the difference of this from the line below?

Reply:
This one is for only Go 1.17 and up.